src/garage/plotter/plotter.py

```diff
@@ -134,6 +147,18 @@ def init_plot(self, env, policy):
         """
         if not Plotter.enable:
             return
+
+        self._env = env
+        self._tf_plot = tf_plot
+        self._rollout = rollout
+        if tf_plot:
```

Comment:
You should be able to put the tensorflow import here (writing it out to a global variable in the process).

Reply:
If TensorFlow is installed, there's virtually no harm in creating a CPU-bound session, even if the user is running PyTorch or NumPy in actuality. Why not eliminate this flag and just create a session if TF is installed?

Reply:
Creating a truly CPU bound session without starting a new process doesn't appear to be possible in TF1. Personally, I'd rather we stop opening TensorFlow sessions in PyTorch experiments if it's easy for us to do so.